src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue24105.cs

```diff
@@ -0,0 +1,24 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue24105 : _IssuesUITest
+	{
+		public override string Issue => "CarouselView doesn't scroll to the right Position after changing the ItemSource with Loop enabled";
+
+		public Issue24105(TestDevice testDevice) : base(testDevice) { }
+		
+		[Test]
+		[Category(UITestCategories.CarouselView)]
+		public void Issue24105Test()
```

Comment:
Use a name here that's more specific to what's being tested. 

These are the names that show up in the test files, so it's useful to have a full description